osquery/tables/system/linux/pci_devices.cpp

```diff
@@ -240,6 +241,121 @@ Status PciDB::getSubsystemInfo(const std::string& vendor_id,
   return Status::success();
 }
 
+Status enrichPCISubsysInfo(Row& row,
+                           std::string& subsystem_attr,
```

Comment:
Would you like to take `subsystem_attr` by value here? As far as it it no output argument.